routing.go

```diff
@@ -21,6 +21,7 @@ import (
 	routing "github.com/libp2p/go-libp2p-routing"
 	notif "github.com/libp2p/go-libp2p-routing/notifications"
 	ropts "github.com/libp2p/go-libp2p-routing/options"
+	"github.com/pkg/errors"
```

Comment:
is this vendored in gx?

Reply:
also, named import please to match the surroundings.

Reply:
Could just import “errors”?

Reply:
Explicitly naming all the packages is not idiomatic. `"github.com/pkg/errors"` is representative of the future direction of error handling in Go, I'm using it here as I'm expecting to need some more advanced error handling for some corners cases in the future.